db/migrations/sqls/20200526142326-add-refresh-tokens-up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE tokens (
+  id serial PRIMARY KEY,
+  token_value varchar(255) NOT NULL UNIQUE
```

Comment:
Since this is the 'tokens' table, could this be renamed to simply 'value' ?

Reply:
+1

Reply:
I tried `value` but I was getting an error from postgres. Will try it again later.

Reply:
Ah value is a reserved keyword (https://www.postgresql.org/docs/8.1/sql-keywords-appendix.html)

Reply:
it's worked when I've changed it...